src/App.js

```diff
@@ -1,23 +1,15 @@
-import logo from './logo.svg';
+
 import './App.css';
+import movieData from './movieData.js';
+import {useState} from 'react'
+import AllMovies from './AllMovies.js'
 
 function App() {
```

Comment:
Something that just occurred to me: should work with ES6 js? I think that would be in line with everything we learned during intermission.